search/templates/search/search-macros.html

```diff
@@ -55,159 +55,40 @@
-                </div>
-                <div class="field">
-                  {{ form.classification.q_finance|safe }}
-                  {{ form.classification.q_finance.label }}
-                </div>
-                <div class="field">
-                  {{ form.classification.statistics|safe }}
-                  {{ form.classification.statistics.label }}
-                </div>
-              </div>
-            </div>
-          </fieldset>
-          <fieldset class="fieldset">
-            <legend class="legend">Date</legend>
-            <div class="field">
-              {{ form.date.all_dates|safe }}
-              {{ form.date.all_dates.label }}
-            </div>
-            <div class="field">
-              {{ form.date.past_12|safe }}
-              {{ form.date.past_12.label }}
-            </div>
-            <div class="field">
-              {{ form.date.specific_year|safe }}
-              {{ form.date.specific_year.label }}
-              <label for="date-year" class="hidden-label">Enter four digit year</label>
-              {{ form.date.year(placeholder="YYYY or YYYY-MM-DD")|safe }}
-            </div>
-            <div class="field">
-              {{ form.date.date_range|safe }}
-              {{ form.date.date_range.label }}
-              {{ form.date.from_date.label }} {{ form.date.from_date(placeholder="YYYY-MM-DD")|safe }}
-              {{ form.date.to_date.label }} {{ form.date.to_date(placeholder="YYYY-MM-DD")|safe }}
-            </div>
-          </fieldset>
-        </section>
-        <section>
-          <div class="level">
-            <div class="level-left">
-              <span class="select is-small">
-                {{ form.results_per_page|safe }}
-              </span>
-               {{ form.results_per_page.label }}
-            </div>
-            <div class="level-right">
-              <button class="button is-link is-large">Search</button>
-            </div>
-          </div>
-        </section>
-        </form>
-      </div>
+{% macro search_results(form, results, metadata, external_url, url_for_page) %}
+<div class="level">
+  <div class="level-left">
+    <div class="level-item">
+        <span class="select is-small">
+          {{ form.results_per_page|safe }}
+        </span>
+        {{ form.results_per_page.label }}
     </div>
-    <div class="column">
-      <div class="box">
-        <h2 class="title is-5">Tips</h2>
-        <p>Text object here for search tips and help? Or just hardcode it?</p>
+
+    <!-- sort select -->
+    <div class="field is-horizontal level-item">
+      <div class="control">
+        <div class="field-label">
+          <label class="label is-small">Sort results by:</label>
+        </div>
+        <div class="select is-small">
+          <select>
+            <option>All fields</option>
+            <option>Author</option>
+            <option>Date</option>
+          </select>
+        </div>
       </div>
-    </div>
```

Comment:
It looks like two <div> openings got taken out but only one </div> removed? Hard to see from this view.

Reply:
![image](https://user-images.githubusercontent.com/3451594/35747306-c7f070c8-0817-11e8-8af1-7fc8123bac86.png)